apps/i18n/common/en_us.json

```diff
@@ -1946,6 +1946,18 @@
   "seeFullQuestion": " ...see full question",
   "seeFullResponse": " ...see full response",
   "seeIncubatorProjects": "See Incubator projects",
+  "sessionLockoutNewAccountHeader": "Just one more thing!",
+  "sessionLockoutPendingHeader": "Thanks! We've contacted your parent.",
+  "sessionLockoutPrompt": "We need your parent to approve your account before you can get started. Please supply us with your parent’s email address so they can grant you permission.",
```

Comment:
Is a provision in the law for a legal guardian (but not a parent) to provide permission? If so do we need language like "parent or guardian"?

Reply:
That is a good question. I noticed this language already used on our existing parent email process, so it seems like a good idea to ask to see if we should just conform to that language.